test/config/generis.conf.php

```diff
@@ -36,11 +36,12 @@
 
 # platform identification
 define('GENERIS_INSTANCE_NAME', '');
-define('GENERIS_SESSION_NAME', '');
+define('GENERIS_SESSION_NAME', 'test');
 
 # paths
 define('ROOT_PATH', '');
-define('ROOT_URL', '');
+// trailing slash is important!
+define('ROOT_URL', 'https://test.com/');
```

Comment:
maybe we can use demo.taotesting.com ? 
we will always have that subdomain and it's free publicity

Reply:
As long as it's a valid domain it doesn't really matter the value. I like it though having a "real" value. Thanks @mike-ionut-mihai-sandu-tao!